digital_ingest_transformation_template.yaml

```diff
@@ -28,35 +29,16 @@ Parameters:
   EfsRootPath:
     Type: String
     Description: Root path at which EFS is mounted.
-  StoragePath:
+  #Is this needed?
+  PackageID: 
     Type: String
-    Description: Location in EFS mount at which source files are stored.
-  TemporaryPath:
-    Type: String
-    Description: Location in EFS mount at which files are unpacked.
-  SourceBucketName:
-    Type: String
-    Description: Name of S3 bucket to which packaged files should be delivered.
-  DigitizationEfsId:
-    Type: String
-    Description: Identifier for digitization delivery EFS.
-  DigitizationEfsAccessPointId:
-    Type: String
-    Description: Identifier for digitization delivery EFS access point.
-  DigitizationRootPath:
-    Type: String
-    Description: Root path for digitized delivery.
-  DigitizationPath:
-    Type: String
-    Description: Path to deliver packages of digitized content for further processing.
-  DigitizationUrl:
-    Type: String
-    Description: URL to post to to trigger further processing for packages of digitized content.
+    Description: String representation of the package identifier
```

Comment:
This is not needed! The lambda which starts the instance of the task supplies this value, since it will be unique to that instance.